services/federator/test/unit/Test/Federator/Options.hs

```diff
@@ -63,6 +68,166 @@ parseFederationStrategy =
     withAllowList =
       AllowList . AllowedDomains . map (either error id . mkDomain)
 
+testSettings :: TestTree
+testSettings =
+  testGroup
+    "settings"
+    [ testCase "parse configuration example (open federation)" $ do
+        assertParsesAs
+          defRunSettings
+          "federationStrategy:\n\
+          \  allowAll:\n\
+          \useSystemCAStore: true",
```

Comment:
Thanks for the tests! Do you think we could use a nicer multi-line string approach here (with that quasi-quoter) so this becomes easier to read?

Reply:
Yep, I switched those multi-line strings to the interpolate quasiquoter.